helper/metricsutil/bucket.go

```diff
@@ -20,7 +20,7 @@ var bucketBoundaries = []struct {
 	{30 * 24 * time.Hour, "30d"},
 }
 
-const overflowBucket = "+Inf"
+const OverflowBucket = "+Inf"
```

Comment:
Instead of `+Inf` do we want do `+30d`?

Reply:
While writing the RFC, I thought about ">30d" or "30d+". I think the main problem I have is that they are visually very similar to the bare "30d", so in a visualization it would be unclear which bucket was which.